pkg/ansible/compress.go

```
@@ -0,0 +1,65 @@
+package ansible
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"io"
+	"os"
+)
+
+
+func Compress(sourceDir string, dest string) error {
+	file, err := os.Open(sourceDir)
+	if err != nil {
+		return err
+	}
+
+	d, _ := os.Create(dest)
```

Please don't ignore errors.

---

Done